force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -355,15 +415,19 @@ public with sharing class ClassesObjectsMethods {
      * @return A list of new Person objects.
      */
     public static List<Person> createMultiplePersons(List<String> names, Integer age) {
-
+        List<Person> newPersons = new List <Person>();
+        for(String listName:names){
+            Person newPerson = new Person(listName,age);
+            newPersons.add(newPerson);
+        }
         // Loop through each name in the provided list of names
             // Create a new Person object for each name
             
             // Add the new Person object to the list of Person objects
         
         
         // Return the list of Person objects
-        return null; // Replace null with the variable you used to store the result
+        return newPersons; // Replace null with the variable you used to store the result
```

Comment:
Great job. The only change I would make, and very minor, is to add tot he list directly. 

```
 public static List<Person> createMultiplePersons(List<String> names, Integer age) {
        List<Person> persons = new List<Person>();
        for (String name : names) {
            persons.add(new Person(name, age));
        }
        return persons;
    }
 ```